ext/LinearSolveHYPRE.jl

```diff
@@ -4,7 +4,7 @@ using HYPRE.LibHYPRE: HYPRE_Complex
 using HYPRE: HYPRE, HYPREMatrix, HYPRESolver, HYPREVector
 using IterativeSolvers: Identity
 using LinearSolve: HYPREAlgorithm, LinearCache, LinearProblem, LinearSolve,
-                   OperatorAssumptions, default_tol, init_cacheval, issquare, set_cacheval
+                   OperatorAssumptions, default_tol, init_cacheval, __issquare, set_cacheval
```

Comment:
we need to be careful with this release.

Reply:
yes. good we have CI test for 1.9

Reply:
tests passing with lts, release, beta locally.